### ISSUE TITLE
Make error. [reference to 'credentials' is ambiguous] 

### ISSUE THREAD

**Issue by conor-byrne** [Microsoft/cpprestsdk#94]
HI guys, wondering if anyone can help, I followed the how to setup, build and run tests on linux in the documentation and had the following problems. I'm working on fedora 23for what it's worth.

cmake works fine, make get's to 46% and then throws up the following:

```
/home/cbyrne/casablanca/Release/tests/functional/http/client/proxy_tests.cpp: In member function ‘void tests::functional::http::client::Suiteproxy_tests::uri_addresshttp_proxy_with_credentialsHelper::RunImpl()’:
/home/cbyrne/casablanca/Release/tests/functional/http/client/proxy_tests.cpp:100:5: error: reference to ‘credentials’ is ambiguous
     credentials cred(U("artur"), U("fred")); // relax, this is not my real password
     ^
In file included from /usr/include/krb5.h:8:0,
                 from /usr/include/openssl/kssl.h:73,
                 from /usr/include/openssl/ssl.h:165,
                 from /usr/include/boost/asio/ssl/detail/openssl_types.hpp:20,
                 from /usr/include/boost/asio/ssl/context_base.hpp:19,
                 from /usr/include/boost/asio/ssl/context.hpp:27,
                 from /usr/include/boost/asio/ssl.hpp:19,
                 from /home/cbyrne/casablanca/Release/include/cpprest/http_client.h:65,
                 from /home/cbyrne/casablanca/Release/tests/functional/http/client/stdafx.h:26,
                 from /home/cbyrne/casablanca/Release/tests/functional/http/client/proxy_tests.cpp:26:
/usr/include/krb5/krb5.h:4182:8: note: candidates are: struct credentials
 struct credentials;
        ^
In file included from /home/cbyrne/casablanca/Release/include/cpprest/http_client.h:50:0,
                 from /home/cbyrne/casablanca/Release/tests/functional/http/client/stdafx.h:26,
                 from /home/cbyrne/casablanca/Release/tests/functional/http/client/proxy_tests.cpp:26:
/home/cbyrne/casablanca/Release/include/cpprest/details/web_utilities.h:81:7: note:                 class web::credentials
 class credentials
       ^
/home/cbyrne/casablanca/Release/tests/functional/http/client/proxy_tests.cpp:101:27: error: ‘cred’ was not declared in this scope
     proxy.set_credentials(cred);
                           ^
tests/functional/http/client/CMakeFiles/httpclient_test.dir/build.make:374: recipe for target 'tests/functional/http/client/CMakeFiles/httpclient_test.dir/proxy_tests.cpp.o' failed
make[2]: *** [tests/functional/http/client/CMakeFiles/httpclient_test.dir/proxy_tests.cpp.o] Error 1
CMakeFiles/Makefile2:471: recipe for target 'tests/functional/http/client/CMakeFiles/httpclient_test.dir/all' failed
make[1]: *** [tests/functional/http/client/CMakeFiles/httpclient_test.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2

```

**Comment by heavyrecoil1974** [comment]
Anyone have any clue? Same error on CentOS 7.2

**Comment by ras0219-msft** [comment]
This should be fixed by 8b6f8325 which is in the development branch.
